docs/core/diagnostics/metrics-instrumentation.md

```diff
@@ -542,6 +547,66 @@ Name                                                  Current Value
 > [!NOTE]
 > OpenTelemetry refers to tags as 'attributes'. These are two different names for the same functionality.
 
+## Using Advice to customize Histogram instruments
+
+Consumers of Histogram instruments can choose different aggregation strategies.
```

Comment:
It might be worth spending a few sentences just to describe what histogram bucketing is. Some readers may not be familiar with them. For example:

```
When using histograms, it is the responsibility of the tool or library collecting the data to decide how best to represent the distribution of values that were recorded. A common strategy is to divide up the range of possible values into sub-ranges called buckets and report how many recorded values were in each bucket. For example a tool might divide numbers into three buckets, those less than 1, those between 1-10, and those greater than 10. If your app recorded the values 0.5, 6, 0.1, 12 then there would be two data points the first bucket, one in the second, and one in the 3rd. Using more buckets with narrower ranges allows representing a distribution with more accuracy, but the tradeoff is that it takes more space to store the results. 
```